tests/test_cmake_build/embedded.cpp

```diff
@@ -0,0 +1,21 @@
+#include <pybind11/embedded.h>
+namespace py = pybind11;
+
+PYBIND11_ADD_EMBEDDED_MODULE(test_cmake_build)(py::module &m) {
+    m.def("add", [](int i, int j) { return i + j; });
```

Comment:
Yeah, eliminating the name repetition is a nice bonus and it also removes a possible beginner's mistake of setting different names for the macro and module (which results in a not-so-obvious error).

As for the argument, a possible alternative is:
```C++
PYBIND11_ADD_EMBEDDED_MODULE(name, m) { ... }
```
But I personally prefer the explicitness of `py::module &m`.